executor/src/witgen/identity_processor.rs

```diff
@@ -135,10 +135,12 @@ impl<'a, 'b, 'c, T: FieldElement, Q: QueryCallback<T>> IdentityProcessor<'a, 'b,
             IdentityKind::Plookup | IdentityKind::Permutation => {
                 self.process_plookup(identity, rows)
             }
-            kind => {
-                unimplemented!(
-                    "Identity of kind {kind:?} is not supported by the identity processor."
-                )
+            IdentityKind::Connect => {
+                // TODO this is not the right cause.
+                Ok(EvalValue::incomplete(IncompleteCause::SolvingFailed))
+                // unimplemented!(
+                //     "Identity of kind {kind:?} is not supported by the identity processor."
+                // )
```

Comment:
Hmm, I'd say this is still an error, no? At least `IdentityProcessor::process_identity` with its current signature will never be able to process it, because it doesn't have access to all cells that might be involved. We could keep it an error and make sure we're filtering connect identities for now.

(BTW, is identity the right term? Should we rename it to constraint?)

Reply:
The idea is that you will update this part in the witgen impl